backend/pkg/msgpack/config.go

```diff
@@ -0,0 +1,37 @@
+package msgpack
+
+import (
+	"fmt"
+)
+
+// Config represents the message pack config.
+type Config struct {
+	Enabled bool `json:"enabled"`
+
+	// TopicNames is a list of topic names that shall be considered for messagepack decoding.
+	// These names can be provided as regex string (e. g. "/.*/" or "/prefix-.*/") or as plain topic name
+	// such as "frontend-activities".
+	// This defaults to `/.*/`
```

Comment:
Either we have to remove this comment line (because right now the default is that enabled msgpack will not be used for any topic names), or we have to add a SetDefaults() method which sets `/.*/` as default. If you choose to use `/.*/` as default the reference config also has to be adapted accordingly; as well as the Validate() method

Reply:
Alright. Added the default /.*/ to match all.